presto-docs/src/main/sphinx/sql/alter-function.rst

```diff
@@ -0,0 +1,47 @@
+==============
+ALTER FUNCTION
+==============
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    ALTER FUNCTION qualified_function_name [ ( parameter_type[, ...] ) ]
+    RETURNS NULL ON NULL INPUT | CALLED ON NULL INPUT
+
+
+Description
+-----------
+
+Alter the definition of an existing function.
+
+Each ``ALTER FUNCTION`` statement can only modify one function
+at a time. If multiple functions are matched by not specifying
+the parameter type list, the query would fail.
```

Comment:
So we support overloading?!

Reply:
We don't. Here is my re-words.
```
Parameter type list must be specified if there are multiple
signatures for the specified function name. Otherwise, the query
would fail.

If exactly one signature exists for the specified function name,
parameter type list might be omitted.
```